apps/web/components/AddressForm/AddressForm.vue

```diff
@@ -14,53 +14,73 @@
     </label>
     <label class="md:col-span-3">
       <UiFormLabel>{{ $t('form.phoneLabel') }}</UiFormLabel>
-      <SfInput name="phone" type="tel" auto-complete="tel" v-model="defaultValues.phone" required />
+      <SfInput
+        name="phone"
+        type="tel"
+        minlength="7"
+        maxlength="15"
```

Comment:
**question**: Are we sure we want this? I didn't see this in the requirements and at least `maxlength` feels counterintuitive to me.